dev/NavigationView/NavigationView.xaml

```diff
@@ -508,10 +508,6 @@
         </Setter>
     </Style>
 
-    <!--Note: Even though UseSystemFocusVisuals is set to True here in the Style, and because
-    bindings in Styles are only evaluated once, we were unable to bring a TemplateSettings.UseSystemFocusVisuals
```

Comment:
>bindings in Styles are only evaluated once [](start = 4, length = 42)

@Chingucoding and @Felix-Dev does this tid bit explain some of the Binding issues you've been seeing in other PRs?

Reply:
That certainly explains the issues I have seen. So the workaround is to essentially handle the binding inside the control logic then?